xml/System/Half.xml

```diff
@@ -211,9 +212,10 @@
         <Parameter Name="value" Type="System.Half" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
+        <param name="value">A <seealso cref="T:System.Half" /> floating-point number.</param>
```

Comment:
Can you bulk change all the `seealso cref` in this file to `see cref`?


```suggestion
        <param name="value">A <see cref="T:System.Half" /> floating-point number.</param>
```

Reply:
Done.